tools/ci_build/github/linux/docker/scripts/install_deps.sh

```diff
@@ -113,8 +113,14 @@ if [ $DEVICE_TYPE = "Normal" ]; then
     ${PYTHON_EXE} -m pip install sympy==1.1.1
 elif [ $DEVICE_TYPE = "gpu" ]; then
     ${PYTHON_EXE} -m pip install sympy==1.1.1
+    # Inferencing pipelines need Torch installed as well to test IOBinding
+    if [ "$PYTHON_VER" = "3.5" ]; then
```

Comment:
This change also requires us move to python 3.6, but our CI build docker image is ubuntu 16.04 which comes with python 3.5.